playwright/e2e/rosa-hosted/rosa-cluster-hosted-private-advanced-creation.spec.ts

```
@@ -0,0 +1,313 @@
+      if (clusterProperties.ClusterAutoscaling.includes('Enabled')) {
+        await expect(createRosaWizardPage.computeNodeRangeLabelValue()).toContainText(
+          `Minimum nodes per machine pool: ${clusterProperties.MachinePools.MiniNodeCount}`,
+        );
+        await expect(createRosaWizardPage.computeNodeRangeLabelValue()).toContainText(
+          `Maximum nodes per machine pool: ${clusterProperties.MachinePools.MaxNodeCount}`,
+        );
+      } else {
+        await createRosaWizardPage.isClusterPropertyMatchesValue(
+          'Compute node count',
+          (
+            parseInt(clusterProperties.MachinePools.NodeCount) *
```

ESLint

[error] Missing radix parameter.

(radix)